test/system/255-auto-update.bats

```diff
@@ -128,6 +141,39 @@ function _confirm_update() {
     _confirm_update $cname $ori_image
 }
 
+@test "podman auto-update - label io.containers.autoupdate=image with rollback" {
+    # Note: the autoupdatebroken image is empty on purpose so it cannot be
+    # executed and force a rollback.  The rollback test for the local policy
+    # is exercising the case where the container doesn't send a ready message.
+    image=quay.io/libpod/autoupdatebroken
+    run_podman pull -q alpine
+    run_podman tag alpine $image
+
+    generate_service autoupdatebroken image
+
+    _wait_service_ready container-$cname.service
+    run_podman auto-update --dry-run --format "{{.Unit}},{{.Image}},{{.Updated}},{{.Policy}}"
+    is "$output" ".*container-$cname.service,$image:latest,pending,registry.*" "Image update is pending."
+
+    run_podman container inspect --format "{{.Image}}" $cname
+    imageID="$output"
+
+    run_podman inspect --format "{{.ID}}" $cname
+    containerID="$output"
+
+    run_podman auto-update --format "{{.Unit}},{{.Image}},{{.Updated}},{{.Policy}}"
+    is "$output" "Trying to pull.*" "Image is updated."
+    is "$output" ".*container-$cname.service,$image:latest,rolled back,registry.*" "Image has been rolled back."
+
+    run_podman container inspect --format "{{.Image}}" $cname
+    is "$output" "$imageID" "container rolled back to previous image"
+
+    run_podman container inspect --format "{{.ID}}" $cname
+    if [[ $output == $containerID ]]; then
+        die "container has not been restarted during rollback (previous id: $containerID, current id: $output)"
+    fi
```

Comment:
Also, as a kindness to test log readers, could you do some cleanup (rm + rmi) here?